CMakeLists.txt

```diff
@@ -5,14 +5,14 @@ project(Fcal2020)
 
 
 # root flags
-set(CMAKE_EXE_LINKER_FLAGS "-L/home/mpotlog/physics-tools/root6.18/build/lib/ -lCore -lRIO -lNet -lHist -lGraf -lGraf3d -lGpad -lTree -lRint -lPostscript -lMatrix -lPhysics -lMathCore -lThread -pthread -lm -ldl -rdynamic")
```

Comment:
Can't compile with your home directory and these flags.

I think all of it unnecessary. I compiled successfully without all these flags. Does it work for you?